### PR TITLE
feat: Exhibitor: Jitsi Integration

### PR DESCRIPTION
**Pull request by MeghalBisht**
as discussed in tuesday meeting

related : #6649 point 4

#### Short description of what this resolves:
Screenshot from 2021-02-19 02-10-03
Screenshot from 2021-02-19 02-09-58
Screenshot from 2021-02-19 02-09-33

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mariobehling** [comment]
Good! Please also change the following

* Every user can see the button, but only logged in users with a ticket should be able to open the video
* If the user joins the exhibitor video rooms it means they are actively participating in the event. For active participants the side panel of the event needs to be active. Therefore the exhibitor video should be in the standard iframe (even though for now the video room is not listed on the left-hand side panel itself).
* Also increase the spacing below the red contact button to the same as above the buttons
* Add a heading above the join video button "Meet Us In Our Exhibitor Video Room"

**Comment by mariobehling** [comment]
If anonymous users or users without a ticket try to join a video room, please show "You need to login or get a ticket for this event before you can join it online."

Screenshot from 2021-02-18 23-39-52

File: app/routes/public/exhibition/video.js [hunk 1]
```diff
@@ -3,17 +3,33 @@ import Route from '@ember/routing/route';
...
 
   renderTemplate() {
     this.render('public/exhibition/video', { into: 'root' });
   }
 
-  model(params) {
-    return this.store.findRecord('exhibitor', params.exhibitor_id, {
+  async model(params) {
+    const event = this.modelFor('public');
+    const exhibitor = this.store.findRecord('exhibitor', params.exhibitor_id, {
```
Comment (iamareebjamal) [inline app/routes/public/exhibition/video.js:20]: You have to await it

File: app/routes/public/exhibition/video.js [hunk 2]
```diff
@@ -3,17 +3,33 @@ import Route from '@ember/routing/route';
...
       include : 'event',
       reload  : true
```
Comment (iamareebjamal) [inline app/routes/public/exhibition/video.js:22]: Why reload?

File: app/routes/public/exhibition/video.js [hunk 3]
```diff
@@ -3,17 +3,33 @@ import Route from '@ember/routing/route';
...
+    const stream = this.store.createRecord('video-stream', {
+      name  : exhibitor.name,
+      event : await event
+    });
+
+    return {
+      exhibitor : await exhibitor,
+      event     : await event,
+      stream    : await stream
+    };
```
Comment (iamareebjamal) [inline app/routes/public/exhibition/video.js:33]: Use rsvp hash

**Comment by mariobehling** [comment]
Please let me know in the chat when this is ready for review and I have a look. Thanks.

**Comment by MeghalBisht** [comment]
Shall i remove the side panel completely? It has blocked the PR and I am not sure how to fix it.

**Review by mariobehling — APPROVED** [review]

**Pull request merged**